### PR TITLE
New changelog page

### PR DESCRIPTION
**Pull request by nanaya**
With new header as well.

Missing pagination for index (if coming at all).

### CODE REVIEW

**Comment by peppy** [comment]
Design:

- Usernames are too weighted. They are the most prominent thing on the page, where they should be something that stays out of way unless someone really cares.
- Category headers are too small. The font size is the same (?) as the content
- Is the "Info" word there for any reason? Should this be omitted if there's only one option present on the page?

Implementation

- For some reason testing this using `php -S` fails to load individual build pages (I think it might be presuming the `.` in the url made it into a local filename?

image

- The page is still one of the slowest to resize on the new web. I think the graph redrawing needs to be worked on to fix this. A resize can take upwards of a second to redraw the page and feels pretty bad from an end-user perspective.

**Comment by arflyte** [comment]
3019f4ae89

Still figuring out nav/breadcrumb

File: resources/assets/coffee/_classes/url.coffee [hunk 1]
```diff
@@ -17,7 +17,37 @@
...
+    'rankings'
+    'session'
+    'store'
+    'users'
+  ].join('|')
+
+
   @beatmapDownloadDirect: (id) -> "osu://dl/#{id}"
 
+  @build: (build) ->
```
[the remark this answers is not in the document]
Comment (nanaya) [reply resources/assets/coffee/_classes/url.coffee]: Not sure what else to call it.

File: resources/assets/less/bem/header-bg.less [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    &::after {
+      .full-size();
+      transition: all 1000ms;
+      content: '';
+      background-color: fade(#222, 80%);
+      opacity: 0;
+      will-change: opacity;
+    }
+
+    &.js-nav2--overlay-active {
```
Comment (notbakaneko) [inline resources/assets/less/bem/header-bg.less]: unused? (also ::after above)

File: resources/assets/coffee/_classes/url.coffee (hunk 1 above)
[the remark this answers is not in the document]
Comment (notbakaneko) [reply resources/assets/coffee/_classes/url.coffee]: it's specifically calling `laroute.route 'changelog.build'` with a build from places that handle build data, so I'm not sure it should even be in here...

**Pull request merged**